scss/components/_itemvehicleweaponsheet.scss

```diff
@@ -1,6 +1,6 @@
-.item-sheet-vehicle-weapon {
-  $swffg-weapon-color: rgb(31, 28, 55);
-  $swffg-weapon-color-light:rgb(204,202,227);
+.item-sheet-weapon {

```

Comment:
Can you clarify the change here? From what I can see, the templates/items/ffg-shipweapon-sheet.html file still has a top level class of item-sheet-vehicle-weapon.